### ISSUE TITLE
What is the chip used to interface the encoder with mcu?

### ISSUE THREAD

**Issue by ArthurDigbySellers** [wiciu15/mitsubishi-encoder#1]
I have a Mitsubishi Servo motor which with a serial encoder which is nearly identical to yours.  I'm trying to figure out how to communicate with it via Serial. More specifically I'm trying to figure out how to use the MR, MRR, MD, and MDR outputs of the encoder.  What chip is used in the photo on the main page and what connections are made? Any insight would be greatly appriciated.

Thanks.

**Comment by wiciu15** [comment]
Full duplex rs485 transceiver, I've been using isl83490/isl83491 but maxim
has similar series. It's important to have a transceiver supporting over
2,5 Mbit speed.
I also managed to get it working with normal half-duplex transceiver MAX485
by shorting MD-MR and MDR-MRR lines. In this case it is critical to have
very fast timing on transceiver DE pin to disable it after transmitting
request byte to not corrupt encoder response. To do that i constantly ask
UART registers if transfer is complete and as soon as it completes I set DE
pin low.


czw., 22 cze 2023, 17:23 użytkownik ArthurDigbySellers <
***@***.***> napisał:
